website/content/events/2025-08-15-cosmic-distance-ladder/_index.de.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Die Kosmische Entfernungsleiter"
+date: 2025-07-15T20:01:28+02:00
+eventStart: 2025-08-15T18:00:00+02:00
+eventEnd: 2025-08-15T20:30:00+02:00
+eventLocation: "Veranstaltungsraum, Haus des Engagements, Rehlingstraße 9, 79100 Freiburg"
+eventGeoLat: 47.98953
+eventGeoLon: 7.83979
+meetupLink: https://www.meetup.com/de-DE/rationality-freiburg/
+lwLink: https://www.lesswrong.com/groups/fFZZ2Ywzsab86EESY
+description: "Wo: Haus des Engagements, Rehlingstraße 9, 79100 Freiburg. Wann: Freitag, 15. August um 18:00 Uhr MESZ."
+eventHost:
+  - Nawid
+eventType:
+  - discussion
+  - presentation
+layout: single
+outputs:
+  - HTML
+  - Calendar
+#url: termine/2025-08-15
```

Comment:
`url: termine/2025-08-15-kosmische-entfernungsleiter` without the hash so it's not commented out